src/main/java/com/spotify/logging/LoggingConfigurator.java

```diff
@@ -109,6 +109,24 @@ public static void configureNoLogging() {
     rootLogger.setLevel(OFF);
   }
 
+  /**
+   * Configure logging with default behaviour for a service. 
+   *
+   * Uses INFO logging level. If the USE_JSON_LOGGING environment variable is set to {@code true}, 
+   * the LogstashEncoder will be used. Otherwise, if the SPOTIFY_SYSLOG_HOST or SPOTIFY_SYSLOG_PORT
+   * environment variable is defined, the syslog appender will be used. Otherwise console appender
+   * will be used.
+   *
+   * @param serviceName  name of the service
+   */
+  public static void configureService(final String serviceName) {
+    if (Boolean.parseBoolean(System.getenv().getOrDefault("USE_JSON_LOGGING", "false"))) {
```

Comment:
Could you please add a unit test to make sure it does not break if we run it with `USE_JSON_LOGGING=YES`

Reply:
Fixed.